### PR TITLE
Update state-dir and example config files

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Updates `state-dir` attribute in the Sensu backend config to to `/var/lib/sensu/sensu-backend`
- Uses new example config files

## Context
https://github.com/sensu/sensu-go/issues/2495

### CODE REVIEW

**Review by amdprophet — APPROVED** [review]
LGTM, but let's have at least one additional review before we merge this.

**Review by cwjohnston — APPROVED** [review]
I think this looks good from a docs perspective and I'm approving it.

@amdprophet did you want another review from eng? I feel like what we are documenting here is different from the [ostensible default](https://github.com/sensu/sensu-go/blob/master/backend/cmd/start.go#L268).

**Comment by apaskulin** [comment]
I'm going to merge this for now, but any further updates are very welcome.

**Pull request merged**
